bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/EntryLogManagerForEntryLogPerLedger.java

```diff
@@ -640,7 +640,9 @@ BufferedLogChannel getCurrentLogForLedgerForAddEntry(long ledgerId, int entrySiz
                 if (logChannel != null) {
                     logChannel.flushAndForceWriteIfRegularFlush(false);
                 }
-                createNewLog(ledgerId);
+                createNewLog(ledgerId,
+                    ": diskFull = " + diskFull + ", allDisksFull = " + allDisksFull
+                        + ", reachEntryLogLimit = " + reachEntryLogLimit);
```

Comment:
we can get to this block even if logChannel is null. so add that aswell to the reason string.

Reply:
add logChannel in the reason string.